README.md

```diff
@@ -4,111 +4,43 @@ An implementation of ripple keypairs & wallet generation using
 [elliptic](https://github.com/indutny/elliptic) which supports rfc6979 and
 eddsa deterministic signatures.
 
-## Generate a random wallet
-```js
-> var generateWallet = require('ripple-keypairs').generateWallet;
-> generateWallet({type: 'ed25519'});
-{ seed: 'sEd7t79mzn2dwy3vvpvRmaaLbLhvme6',
-  accountID: 'r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ',
-  publicKey: 'ED5F5AC8B98974A3CA843326D9B88CEBD0560177B973EE0B149F782CFAA06DC66A' }
-```
+## API Methods
 
-## Derive a wallet from a seed
-```js
-> var walletFromSeed = require('ripple-keypairs').walletFromSeed;
-> walletFromSeed('sEd7t79mzn2dwy3vvpvRmaaLbLhvme6');
-{ seed: 'sEd7t79mzn2dwy3vvpvRmaaLbLhvme6',
-  accountID: 'r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ',
-  publicKey: 'ED5F5AC8B98974A3CA843326D9B88CEBD0560177B973EE0B149F782CFAA06DC66A' }')
 ```
-
-## Generate random validator keys
-```js
-> var generateValidatorKeys = require('ripple-keypairs').generateValidatorKeys;
-> generateValidatorKeys();
-{ seed: 'ssC7Y9LMKhuzFMVueaj2fnTuGLftA',
-  publicKey: 'n9MU2RsULUayZnWeLssjbMzVRPeVUUMgiPYTwe8eMgpdGDWp5t8C' }
+generateSeed({entropy?: Array<integer>, algorithm?: string}) -> string
 ```
+Generate a seed that can be used to generate keypairs. Entropy can be provided as an array of 32-bit integers. If provided, it must be at least 16 words long. If not provided, entropy will be automatically generated. The "algorithm" defaults to "secp256k1", but can also be set to "ed25519". The result is a seed encoded in base58, starting with "s".
 
-## Derive validator keys from a seed
-```js
-> var validatorKeysFromSeed = require('ripple-keypairs').validatorKeysFromSeed;
-> validatorKeysFromSeed('ssC7Y9LMKhuzFMVueaj2fnTuGLftA');
-{ seed: 'ssC7Y9LMKhuzFMVueaj2fnTuGLftA',
-  publicKey: 'n9MU2RsULUayZnWeLssjbMzVRPeVUUMgiPYTwe8eMgpdGDWp5t8C' }
 ```
-
-## Derive accountID matching a validator public key (aka public generator)
-```js
-> var nodePublicAccountID = require('ripple-keypairs').nodePublicAccountID;
-> nodePublicAccountID('n9MXXueo837zYH36DvMc13BwHcqtfAWNJY5czWVbp7uYTj7x17TH')
-'rhcfR9Cg98qCxHpCcPBmMonbDBXo84wyTn'
+deriveKeypair(seed: string) -> {privateKey: string, publicKey: string}
 ```
+Derive a public and private key from a seed. The keys are represented as hexadecimal strings. A single metadata byte is prepended so the keys are 33 bytes long.
```

Comment:
Lol, well I guess that ("33 bytes") leaves enough info for the alert and
curious. It might be juuuuust the right amount of info. Was that the
intention?